app/liquid_tags/user_tag.rb

```diff
@@ -1,82 +1,33 @@
 class UserTag < LiquidTagBase
   include ApplicationHelper
   include ActionView::Helpers::TagHelper
-  attr_reader :user
+  PARTIAL = "users/liquid".freeze
 
   def initialize(_tag_name, user, _tokens)
```

Comment:
The `initialize` method is never closed.

Reply:
just updated this file. thanks for pointing this out! 😃